src/frontend/js/toggle-dashboard-section.ts

```diff
@@ -18,3 +21,96 @@ async function toggleDashboardSection({ target }: Event): Promise<void> {
   content.classList.toggle("max-h-160");
   content.classList.toggle("max-h-0");
 }
+
+
+async function loadRssFeed() {
+  const feedWidget = document.querySelector(".rss-feed");
+
+  if (feedWidget) {
+    const feedLoading = document.getElementById("feed-loading");
+    const reloadButton = document.getElementById("reload");
+
+    const handleError = () => {
+      feedLoading.classList.add("hidden");
+      reloadButton.classList.remove("hidden");
+      reloadButton.addEventListener("click", loadRssFeed);
+    };
+    
+    feedLoading.classList.remove("hidden");
+    reloadButton.classList.add("hidden");
+    const url = feedWidget.attributes[1].value;
+
+    // set timeout
+    const controller = new AbortController();
+    const timeoutId = setTimeout(() => {
+      controller.abort();
+      handleError();
+    }, 5000);
+
+    fetch(url, {signal: controller.signal}).then(async (response: Response) => {
+      if (response.status === 200 && response.ok) {
+        feedLoading.classList.add("hidden");
+        clearTimeout(timeoutId);
+        let parser = new DOMParser;
+        let feedHtml = parser.parseFromString(
+          parser.parseFromString(await response.text(), "text/xml").activeElement.innerHTML, "text/html");
```

Comment:
On firefox everything is fine, but on chrome the inner call of `parseFromString` returns a xmlDocument with activeElement set to null which throws a TypeError, which you can see if you pass an argument called `error` to the catch function.
For a solution I could found one common element of the xmlDocument (set in chrome and in firefox), which is called `documentElement`, which we could use here instead.
Can you confirm the problem on chrome?

Reply:
No worries as you are absolutely right again. I can confirm the issue and your proposed solution seems to work perfectly fine in chrome as well as in firefox. Thanks again!